### PR TITLE
usm: Added docker-swarm documentation

### PR DESCRIPTION
**Pull request by guyarb**
### What does this PR do?

Adds docker-swarm installation instructions for USM

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/universal_service_monitoring/_index.md [hunk 1]
```diff
@@ -374,6 +374,16 @@ services:
      - /:/host/root:ro
 ```
 
+{{% /tab %}}
+{{% tab "Docker Swarm" %}}
+
+As `Docker Swarm` does not yet support of changing `security_opt`, then we can run only if the Operating System
+does not have a running `apparmor` instance.
```
Comment (buraizu) [inline content/en/universal_service_monitoring/_index.md:381]:
```suggestion
As `Docker Swarm` does not yet support the changing of `security_opt`, the operating system
must not have a running `apparmor` instance.
```

File: content/en/universal_service_monitoring/_index.md [hunk 2]
```diff
@@ -374,6 +374,16 @@ services:
...
+
+If that is the case, then use the same `docker-compose.yml` file from `Docker-Compose` [section][1] besides the field `security_opt`.
```
Comment (buraizu) [inline content/en/universal_service_monitoring/_index.md:383]:
```suggestion
If the operating system does not have a running `apparmor` instance, then use the same `docker-compose.yml` file from `Docker-Compose` [section][1] beside the field `security_opt`.
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, just suggested some minor edits.

**Comment by guyarb** [comment]
@buraizu committed the suggestions, thanks

**Review by buraizu — APPROVED** [review]
Thanks for applying those changes, I left one last small suggestion but overall this LGTM

**Pull request merged**